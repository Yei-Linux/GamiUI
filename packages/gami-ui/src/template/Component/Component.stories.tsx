import React from 'react'
import { Meta } from '@storybook/react'

import Component from '.'

export default {
  title: 'Atomic/Component',
  component: Component,
  args: {},
  argTypes: {},
} as Meta

export const Basic = (args: any) => <Component {...args} />
Basic.args = {}
