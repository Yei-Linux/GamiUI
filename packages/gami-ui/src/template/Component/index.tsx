export { default } from './Component'
export { options } from './constants'
