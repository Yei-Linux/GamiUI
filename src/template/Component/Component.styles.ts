import styled from '@emotion/styled'

export const ComponentWrapper = styled.div``
