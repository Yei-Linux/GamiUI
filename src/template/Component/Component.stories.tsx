import React from 'react'
import { Meta } from '@storybook/react'

import Component from '.'

export default {
  title: 'Atomic/Component',
  component: Component,
  args: { ...Component.defaultProps },
  argTypes: {},
} as Meta

export const Basic = () => <Component />
Basic.args = {}
