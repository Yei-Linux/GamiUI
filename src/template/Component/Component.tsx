import React from 'react'
import * as S from './Component.styles'

export interface IComponent {
  /**
   * Prop Of Component
   */
  prop?: any
}

const Component = ({ prop }: IComponent) => {
  return <S.Component />
}

export default Component
