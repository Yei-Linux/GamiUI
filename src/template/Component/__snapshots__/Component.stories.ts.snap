// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Atomic/Component Basic 1`] = `
<div
  className="css-92d30y-ComponentWrapper eksn8810"
/>
`;
